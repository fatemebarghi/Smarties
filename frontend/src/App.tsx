import React from "react";
import SongList from "./components/songList/SongList";
import Player from "./components/player/Player";
import { Navbar, Container } from "react-bootstrap";
import "bootstrap/dist/css/bootstrap.min.css";
import "./App.css";

function App() {
  return (
    <div className="App">
      <Navbar bg="light" className="mb-3">
        <Container>
          <Navbar.Brand href="#home">
            <b>Jam Community</b>
          </Navbar.Brand>
        </Container>
      </Navbar>
      <Container>
        <SongList />
      </Container>
      <Player />
    </div>
  );
}

export default App;
