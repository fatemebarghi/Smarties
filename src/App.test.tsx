
// import { render } from '@testing-library/react';
import { shallow, mount, render } from 'enzyme';
import App from './App';

describe('renders <App />', () => {
  it('should pass shallow snapshot test', () => {
    const wrapper = render(<App />);
    expect(wrapper).toMatchSnapshot();
  });
});