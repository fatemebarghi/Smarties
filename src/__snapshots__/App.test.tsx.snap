// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders <App /> should pass shallow snapshot test 1`] = `
LoadedCheerio {
  "0": <div
    class="App"
  >
    <nav
      class="mb-3 navbar navbar-expand navbar-light bg-light"
    >
      <div
        class="container"
      >
        <a
          class="navbar-brand"
          href="#home"
        >
          <b>
            Jam Community
          </b>
        </a>
      </div>
    </nav>
    <div
      class="container"
    >
      <div
        data-testid="song-list"
      />
    </div>
  </div>,
  "_root": LoadedCheerio {
    "0": Document {
      "children": Array [
        <html>
          <head />
          <body />
        </html>,
      ],
      "endIndex": null,
      "next": null,
      "parent": null,
      "prev": null,
      "startIndex": null,
      "type": "root",
      "x-mode": "quirks",
    },
    "_root": [Circular],
    "length": 1,
    "options": Object {
      "decodeEntities": true,
      "xml": false,
    },
  },
  "length": 1,
  "options": Object {
    "decodeEntities": true,
    "xml": false,
  },
}
`;
